apps/web/pages/event-types/index.tsx

```diff
@@ -69,9 +69,7 @@ const Item = ({
 }) => {
   const { t } = useLocale();
 
-  function isCalendarConnectedMissing() {
-    return connectedCalendars?.length && !type.team && !type.destinationCalendar;
-  }
+  const isCalendarConnectedMissing = connectedCalendars?.length && !type.team && !type.destinationCalendar;
```

Comment:
Fixes 0 issue appearing. Don't think this needed to be a function as it was being called on render anyway.